compute_tools/src/compute.rs

```diff
@@ -1367,6 +1368,38 @@ LIMIT 100",
         download_size
     }
 
+    pub fn install_extension(&self, ext_name: &str, db_name: &str) -> Result<String> {
+        let mut conf =
+            Config::from_str(self.connstr.as_str()).context("Failed to parse connection string")?;
+        conf.dbname(db_name);
+
+        let mut db_client = conf
+            .connect(NoTls)
+            .context("Failed to connect to the database")?;
+
+        let query = format!(
+            "CREATE EXTENSION IF NOT EXISTS {}",
+            ext_name.to_string().pg_quote()
+        );
+        info!("creating extension with query: {}", query);
+
+        db_client
+            .execute(&query, &[])
+            .context(format!("Failed to execute query: {}", query))?;
+
+        let version_query = format!(
+            "SELECT extversion FROM pg_extension WHERE extname = '{}'",
+            ext_name.to_string().pg_quote()
+        );
```

Comment:
Let's use a paramaterized query just to be safe.

Reply:
I assume you mean for all of the queries? `simple_query` doesn't appear to allow that? Many other queries in the same file are done first by formatting the query string as here and then using `simple_query`.